.github/workflows/php.yml

```
@@ -22,6 +22,7 @@ jobs:
         project: [
           'Aws',
           'Context/Swoole',
+          'Instrumentation/Curl',
           'Instrumentation/CakePHP',
           'Instrumentation/CodeIgniter',
           'Instrumentation/ExtAmqp',
```

we just merged a PR to sort these, so let's stick to that :)
```suggestion
          'Instrumentation/CakePHP',
          'Instrumentation/CodeIgniter',
          'Instrumentation/Curl',
          'Instrumentation/ExtAmqp',
```